src/BlockEntities/NoteEntity.h

```diff
@@ -7,7 +7,7 @@
 
 
 
-enum ENUM_NOTE_INSTRUMENTS
+enum ENUm_Pitch_INSTRUMENTS
```

Comment:
Why rename this in such a weird fashion?

Reply:
This is a good Question - didn't notice that.. used the rename tool and this was the result - gonna change that

Reply:
reverted changes